var/spack/repos/builtin/packages/mesa-glu/package.py

```diff
@@ -34,4 +34,5 @@ class MesaGlu(AutotoolsPackage):
 
     version('9.0.0', 'bbc57d4fe3bd3fb095bdbef6fcb977c4')
 
-    depends_on('mesa')
+    variant('mesa', default=True,  description='Depends on mesa')
```

Comment:
You might want a more detailed description. Aside from the dependency, what difference is there between `mesa-glu+mesa` and `mesa-glu~mesa`? It isn't clear from the description.